src/style_alt.css

```diff
@@ -16,7 +16,6 @@
     font-weight: 500;
```

Comment:
Minor, this file is not being used, can actually be removed

Reply:
Will do